packages/manager/apps/container/src/container/nav-reshuffle/onboarding/style.module.scss

```diff
@@ -17,8 +17,8 @@
 }
 
 .welcomePopover {
-  width: 315px;
-  max-width: 315px !important;
+  width: 360px;
+  max-width: 360px !important;
```

Comment:
This was not what I meant, I wanted to ask if there is a way to avoid setting `width` and `max-width` with hardcoded px values.
From what I see from the ticket, there was no request to resize it, so I guess it was done to fit the content inside the popup, but I think there should be a way to let the popup define it's own size depending on its content

Reply:
the oui-popover has a fixed width of 300px, the content gets truncated. Therefore, I had no choice but to maintain a fixed size with max-width: 360px

Reply:
Ok so just switch to 22.5rem